content/hi/docs/home/_index.md

```diff
@@ -55,7 +55,7 @@ cards:
   button: प्रलेखन में योगदान करें 
   button_path: /docs/contribute
 - name: release-notes
-  title: कुबेरनेट्स रिलीज नोट्स 
+  title: कुबेरनेट्स डाउनलोड करें
   description: यदि आप कुबेरनेट्स इंस्टॉल कर रहे हैं या नवीनतम संस्करण में अपग्रेड कर रहे हैं, तो वर्तमान रिलीज़ नोट्स देखें।
   button: कुबेरनेट्स डाउनलोड करें
   button_path: "/docs/setup/release/notes"
```

Comment:
Considering the title change, should the `button_path` be also changed to align with this change?

Reply:
thanks for pointing that out, i'll update the button path too !

Reply:
hi @dipesh-rawat , I've updated `button_path` but the download page is still in English language.

Reply:
In that case, the specific page could benefit from localization in Hindi. Once that's done, we can revisit this page and add 'hi' to the link.